src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/app.config

```diff
@@ -3,6 +3,10 @@
   <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
+        <assemblyIdentity name="Microsoft.WindowsAzure.Storage" publicKeyToken="31bf3856ad364e35" culture="neutral" />
```

Comment:
Not the same @SimonCropp, [range/versions are different](https://github.com/Particular/NServiceBus.Persistence.AzureStorage/pull/140/files#diff-72ec67f87259888d5ed887536235f337R22) 😛 

> "0.0.0.0-6.2.2.0" newVersion="6.2.2.0"